### ISSUE TITLE
Not hide browser window

### ISSUE THREAD

**Issue by Sanders-and-Sanders** [QuiteRSS/quiterss#1377]
If the Classic layout is installed, then the browser window cannot be closed. In earlier versions it worked, now it doesn't. Please make the option to close the browser window in the Classic layout. Newspaper layout is not suitable for me.

**Comment by alexdevero** [comment]
I have the same issue. The browser window cannot be closed.

![quiterss](https://user-images.githubusercontent.com/4237724/100447994-dec92180-30b1-11eb-9e66-f41ade74b3a7.PNG)

**Comment by Sanders-and-Sanders** [comment]
Yes, many do not use the built-in browser. It is more convenient to open on the outside. But this window, unfortunately, is still not removed and is terribly in the way.

**Comment by dsalt** [comment]
I'd not had this problem until today, when I poked at the configuration – and, suddenly, there's this annoying built-in browser pane which just **won't go away**.
